mmv1/third_party/terraform/utils/common_diff_suppress.go.erb

```diff
@@ -184,6 +184,14 @@ func durationDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return oDuration == nDuration
 }
 
+// returns true if old=new or old='auto'
+func compareSelfOrAuto(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
Since this func is very specific to redis instances, could you rename it to `RedisInstanceSecondaryIpRangeDiffSuppress` and move it to resource_redis_instance.go? (and move the tests as well.)

Reply:
Moved the changes to mmv1/templates/terraform/constants/redis_instance.go